backend/src/LibBackend/SqlCompiler.fs

```diff
@@ -260,10 +261,24 @@ let rec lambdaToSql
   (expr : Expr)
   : string * List<string * SqlValue> * TypeReference =
   let lts (expectedType : TypeReference) (expr : Expr) =
-    lambdaToSql fns types symtable paramName dbTypeRef expectedType expr
+    lambdaToSql fns constants types symtable paramName dbTypeRef expectedType expr
 
   let (sql, vars, actualType) =
     match expr with
+    | EConstant(_, (FQConstantName.Stdlib name as fqName)) ->
+      let nameStr = FQConstantName.toString fqName
+      match Map.get name constants with
+      | Some c ->
+        typecheck nameStr c.returnType expectedType
+        let random = randomString 8
+        let newname = $"{nameStr}_{random}"
+        // let! constant = c.constant
+        let (sqlValue, actualType) = dvalToSql expectedType c.constant.Result
+        $"(@{newname})", [ newname, sqlValue ], actualType
+      | None ->
+        error
+          $"Only builtin constants can be used in queries right now; {nameStr} is not a builtin constant"
```

Comment:
It seems like we should be able to use non-builtin constants, but if that's hard to swing then that can be something to follow up with.